bootstrap/js/contact.js

```diff
@@ -0,0 +1,76 @@
+//Add an event listener for the form submission to create a new user
+
+const form = document.querySelector("form");
+
+
+const formEvent = form.addEventListener('submit', event => {
+   event.preventDefault(); 
+   const username = document.querySelector("#username").value;
+   const email = document.querySelector("#email").value;
+   const message = document.querySelector("#message").value;
+
+   const user = { username, email, message };
+
+
+   createUser(user);
+});
+
+
+
+const createUser = (user) => {
+    let postObject = {...user}
+axios({
+    method: 'post',
+    url: ' https://gg-web-api.herokuapp.com/users/',
+    data: {
+      "username": `${postObject.username}`,
+      "email": `${postObject.email}`,
+      "message": `${postObject.message}`
+    }
+  });
+
+}
+
+createUser();
```

Comment:
I'm not sure why you're calling createUser() here. Can you share the reason, please?

Reply:
there is no particular reason as to why I am calling that function, in my thinking, correct me if I'm wrong, I am trying to invoke the function by calling the createUser().